specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-Preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,1252 @@
+        "operationId" : "/entities/recognition/general",
+        "consumes" : [ "application/json", "text/json" ],
+        "produces" : [ "application/json", "text/json" ],
+        "parameters" : [ {
+          "name" : "model-version",
+          "in" : "query",
+          "description" : "(Optional) This value indicates which model will be used for scoring. If a model-version is not specified, the API should default to the latest, non-preview version. ",
+          "type" : "string"
+        }, {
+          "name" : "showStats",
+          "in" : "query",
+          "description" : "(Optional) if set to true, response will contain input and document level statistics.",
+          "type" : "boolean"
+        }, {
+          "in" : "body",
+          "name" : "multiLanguageBatchInput",
+          "description" : "Collection of documents to analyze.",
+          "required": true,
+          "schema" : {
+            "$ref" : "#/definitions/MultiLanguageBatchInput"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "A successful call results in a list of recognized entities returned for each valid document.",
+            "examples" : {
+              "application/json" : {
+                "documents" : [ {
+                  "id" : "1",
+                  "entities" : [ {
+                    "text" : "Microsoft",
+                    "type" : "Organization",
+                    "offset" : 25,
+                    "length" : 9,
+                    "score" : 0.99
+                  } ]
+                }, {
+                  "id" : "2",
+                  "entities" : [ {
+                    "text" : "John",
+                    "type" : "Person",
+                    "offset" : 25,
+                    "length" : 9,
+                    "score" : 0.99
+                  }, {
+                    "text" : 20,
+                    "type" : "Number",
+                    "subType" : "Age",
+                    "offset" : 30,
+                    "length" : 2,
+                    "score" : 0.88
+                  } ]
+                } ],
+                "errors" : [ ],
+                "modelVersion": "2019-10-01"
+              }
+            },
+            "schema" : {
+              "$ref": "#/definitions/EntitiesResult"
+            }
+          },
+          "500" : {
+            "description" : "Error Response",
+            "schema" : {
+              "$ref" : "#/definitions/Error"
+            }
+          }
+        }
+      }
+    },
+    "/entities/recognition/pii" : {
+      "post" : {
+        "summary" : "Entities containing personal information",
+        "description" : "The API returns a list of personal information entities (\\\"SSN\\\", \\\"Bank Account\\\" etc) in the document. See the &lt;a href=\"https://aka.ms/talangs\"&gt;Supported languages in Text Analytics API&lt;/a&gt; for the list of enabled languages.\n",
+        "operationId" : "/entities/recognition/pii",
```

Comment:
Please use an appropriate name for method name. It shows up in the generated Java code:
```Java
Mono<SimpleResponse<Object>> /entities/recognition/general(@HostParam("Endpoint") String endpoint, @QueryParam("model-version") String modelVersion, @QueryParam("showStats") Boolean showStats, @BodyParam("application/json; charset=utf-8") MultiLanguageBatchInput multiLanguageBatchInput);
```
Might be `RecognitionPII` could be a better name. A better naming pattern is to capitalize the first letter of word and no space between words.